### PR TITLE
Moving some more methods away from daemon.go

### PR DESCRIPTION
**Pull request by vdemeester**
Missed some more in my previous pull-request :sweat_smile: 🐙. 

`daemon.go` is still 957 lines long with pretty big methods but at least almost all methods there are for daemon only.

/cc @thaJeztah @runcom @LK4D4 

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

### CODE REVIEW

File: daemon/daemon.go [hunk 1]
```diff
@@ -317,35 +310,6 @@ func (daemon *Daemon) waitForNetworks(c *container.Container) {
...
-
-func (daemon *Daemon) getEntrypointAndArgs(configEntrypoint strslice.StrSlice, configCmd strslice.StrSlice) (string, []string) {
-	if len(configEntrypoint) != 0 {
-		return configEntrypoint[0], append(configEntrypoint[1:], configCmd...)
-	}
-	return configCmd[0], configCmd[1:]
-}
-
-// GetLabels for a container or image id
-func (daemon *Daemon) GetLabels(id string) map[string]string {
```
Comment (vdemeester) [inline daemon/daemon.go]: This method is completely removed, not used anymore :sweat_smile:

File: daemon/stats_collector.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package daemon
```
Comment (LK4D4) [inline daemon/stats_collector.go]: this probably should become just `stats.go`

Comment (vdemeester) [inline daemon/stats_collector.go]: good point 😇

**Comment by vdemeester** [comment]
@LK4D4 updated

**Comment by crosbymichael** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**